-------------------------------------------------------------------------------
Test set: br.com.boletos.v1.service.BoletoServiceTest
-------------------------------------------------------------------------------
Tests run: 8, Failures: 4, Errors: 0, Skipped: 0, Time elapsed: 0.04 s <<< FAILURE! - in br.com.boletos.v1.service.BoletoServiceTest
deveValidarSeVencimentoBoletoAntesDataAtual  Time elapsed: 0.006 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Unexpected exception type thrown, expected: <br.com.boletos.exceptions.DataVencimentoAntesDataAtualException> but was: <java.lang.NullPointerException>
	at br.com.boletos.v1.service.BoletoServiceTest.deveValidarSeVencimentoBoletoAntesDataAtual(BoletoServiceTest.java:155)
Caused by: java.lang.NullPointerException
	at br.com.boletos.v1.service.BoletoServiceTest.lambda$deveValidarSeVencimentoBoletoAntesDataAtual$3(BoletoServiceTest.java:155)
	at br.com.boletos.v1.service.BoletoServiceTest.deveValidarSeVencimentoBoletoAntesDataAtual(BoletoServiceTest.java:155)

deveValidarAssociado  Time elapsed: 0.003 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Expected br.com.boletos.exceptions.AssociadoNaoExisteNaAPIException to be thrown, but nothing was thrown.
	at br.com.boletos.v1.service.BoletoServiceTest.deveValidarAssociado(BoletoServiceTest.java:112)

deveValidarSeBoletoJaFoiPago  Time elapsed: 0.003 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Unexpected exception type thrown, expected: <br.com.boletos.exceptions.BoletoPagoException> but was: <java.lang.NullPointerException>
	at br.com.boletos.v1.service.BoletoServiceTest.deveValidarSeBoletoJaFoiPago(BoletoServiceTest.java:142)
Caused by: java.lang.NullPointerException
	at br.com.boletos.v1.service.BoletoServiceTest.lambda$deveValidarSeBoletoJaFoiPago$2(BoletoServiceTest.java:142)
	at br.com.boletos.v1.service.BoletoServiceTest.deveValidarSeBoletoJaFoiPago(BoletoServiceTest.java:142)

deveValidarSeValorBoletoEDivergente  Time elapsed: 0.003 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Unexpected exception type thrown, expected: <br.com.boletos.exceptions.ValorDeBoletoDivergenteNoPagamentoException> but was: <java.lang.NullPointerException>
	at br.com.boletos.v1.service.BoletoServiceTest.deveValidarSeValorBoletoEDivergente(BoletoServiceTest.java:127)
Caused by: java.lang.NullPointerException
	at br.com.boletos.v1.service.BoletoServiceTest.lambda$deveValidarSeValorBoletoEDivergente$1(BoletoServiceTest.java:127)
	at br.com.boletos.v1.service.BoletoServiceTest.deveValidarSeValorBoletoEDivergente(BoletoServiceTest.java:127)

